dash/src/util/LocalityDomain.cc

```diff
@@ -81,18 +75,20 @@ dash::util::LocalityDomain::LocalityDomain(
 
 dash::util::LocalityDomain::~LocalityDomain()
 {
-  DASH_LOG_TRACE_VAR("LocalityDomain.~()", _domain->domain_tag);
+  DASH_LOG_TRACE_VAR("LocalityDomain.~()", _domain);
 
-  if (_subdomains != nullptr) {
+  if (nullptr != _subdomains) {
```

Comment:
Aside from the introduced Yoda Condition, the check for nullptr is not strictly required here...